### PR TITLE
fix for skimage deprecated import in acstools

### PR DESCRIPTION
**Pull request by SaraOgaz**
Reported by @dborncamp.

### CODE REVIEW

**Comment by dborncamp** [comment]
It would be good to note that this came in because the testing environment for this was UREKA SSBX which used scikit image version .10. Thank you for fixing that!

File: lib/acstools/satdet.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
     Only tested for ACS/WFC FLT and FLC images, but it should
     theoretically work for any instrument.
 
+    Requires skimage version 0.11.x or later
```
Comment (pllim) [inline lib/acstools/satdet.py]: Nitpicking: End the sentence with a period.

File: lib/acstools/satdet.py [hunk 2]
```diff
@@ -141,10 +144,11 @@
 from astropy.stats import biweight_midvariance, sigma_clipped_stats
 from astropy.utils.exceptions import AstropyUserWarning
 from scipy import stats
-from skimage import filter as filt
+from skimage import filters as filt
```
Comment (pllim) [inline lib/acstools/satdet.py]: This import is not necessary anymore?

File: setup.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = acstools
-version = 2.0.1
+version = 2.0.2
```
Comment (pllim) [inline setup.cfg]: This should also be updated in `defsetup.py` for consistency although I don't think that file is used.

**Comment by pllim** [comment]
p.s. It is "deprecated" :wink:

**Comment by pllim** [comment]
p.p.s. If you want to practice squashing commits to increase your git-fu, you can use this as a practice to squash all your commits here into one. Not that it is necessary here, but just if you want some practice.

**Comment by pllim** [comment]
Once the comments are addressed, I will merge this and make a release.

**Comment by SaraOgaz** [comment]
Thanks @pllim!  I should be able to get to these comments on Friday.

**Comment by SaraOgaz** [comment]
Closing this pull request, wasn't using a new branch *DOH*

**Pull request closed without merging**